velox/functions/sparksql/SparkPartitionId.cpp

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "velox/functions/sparksql/SparkPartitionId.h"
+
+namespace facebook::velox::functions::sparksql {
+namespace {
+
+class SparkPartitionId final : public exec::VectorFunction {
+ public:
+  explicit SparkPartitionId(int32_t partitionId) : partitionId_(partitionId) {}
+
+  void apply(
+      const SelectivityVector& rows,
+      std::vector<VectorPtr>& /*args*/,
+      const TypePtr& outputType,
+      exec::EvalCtx& context,
+      VectorPtr& result) const override {
+    auto localResult = BaseVector::createConstant(
+        outputType, partitionId_, rows.end(), context.pool());
+    context.moveOrCopyResult(localResult, rows, result);
+  }
+
+  bool isDefaultNullBehavior() const override {
```

Comment:
This can be true since this function has no arguments.

Reply:
Updated, thanks.